proteuslib/tools/parameter_sweep.py

```diff
@@ -131,6 +146,15 @@ def _build_combinations(d, sampling_type, num_samples, comm, rank, num_procs):
             sorting = np.argsort(param_values[0])
             global_combo_array = np.vstack(param_values).T
             global_combo_array = global_combo_array[sorting, :]
+
+        elif sampling_type == SamplingType.RANDOM_LHS:
+            lb = [val[0] for val in param_values]
+            ub = [val[1] for val in param_values]
+            lhs = sampling.LatinHypercubeSampling([lb, ub], number_of_samples=num_samples, sampling_type='creation')
```

Comment:
If we add more of the PySMO sampling methods, we should consider if there's a more general approach.